tests/containers/workbenches/jupyterlab/libraries_test.py

```
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import pathlib
+from typing import TYPE_CHECKING
+
+from tests.containers import docker_utils
+from tests.containers.conftest import get_image_metadata, Image
```

Ruff (0.12.2)

`tests.containers.conftest.get_image_metadata` imported but unused

Remove unused import: `tests.containers.conftest.get_image_metadata`

(F401)